### PR TITLE
1-Click Run

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
### Description

This PR adds some docker files that allow a user to run datatools with just one click!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +30,7 @@ Note: `dev` branch docs (which refer to the default `branch` and are more up-to-
 
 ## Getting in touch
 
-We have a Gitter [space](https://matrix.to/#/#transit-data-tools:gitter.im) for the full TRANSIT-Data-Tools project where you can post questions and comments.
```
Comment (philip-cline) [inline README.md:19]: Revert this change please

**Review by philip-cline — APPROVED** [review]
Just *one* click???

**Review by amy-corson-ibigroup — APPROVED** [review]
one CLICK  🎉 ONE CLICK  🎉  **ONE CLICK**  🎉

**Comment by miles-grant-ibigroup** [comment]
Blocked wating for [this](https://github.com/ibi-group/datatools-server/pull/554) to me merged
